docs/UserGuide.adoc

```diff
@@ -333,6 +333,18 @@ Please ensure that the unit specified corresponds accordingly.
 _(If your inventory had 500 g of milk, it will be subtracted and updated to 300 g of butter)_
 |===
 
+==== Remove all ingredients from the inventory.
+This command allows you to remove all ingredients from the inventory.
+
+- Format: `inventory clear ingredient`
```

Comment:
For the clear commands, have we decided that it will simply be `cart/inventory clear`?

Reply:
The updated one was changed to `cart clear ingredient`